yarn.lock

```diff
@@ -3654,11 +3654,16 @@ path-parse@^1.0.7:
   resolved "https://registry.npmjs.org/path-parse/-/path-parse-1.0.7.tgz"
   integrity sha512-LDJzPVEEEPR+y48z93A0Ed0yXb8pAByGWo/k5YYdYgpY2/2EsOsksJrq7lOHxryrVOn1ejG6oAp8ahvOIQD8sw==
 
-path-to-regexp@0.1.7, path-to-regexp@^0.1.2:
+path-to-regexp@0.1.7:
   version "0.1.7"
```

Comment:
well 0.1.7 is still vulnerable according to the advisory no ?

Reply:
In theory yes, but this is part of a dev-dependency, so not critical and not blocking CI

Reply:
Isn't this a dep of express ?

Reply:
which is a dev dep obviously nvm :facepalm: